docs/source/common/lightning_module.rst

```diff
@@ -1364,12 +1364,18 @@ load_from_checkpoint
 on_hpc_save
 ~~~~~~~~~~~
 
+.. warning:: ``on_hpc_save`` is deprecated in v1.6 and will be removed in v1.8.
+    Please use ``model.on_save_checkpoint()`` instead.
+
 .. automethod:: pytorch_lightning.core.lightning.LightningModule.on_hpc_save
     :noindex:
 
 on_hpc_load
 ~~~~~~~~~~~
 
+.. warning:: ``on_hpc_load`` is deprecated in v1.6 and will be removed in v1.8.
+    Please use ``model.on_load_checkpoint()`` instead.
```

Comment:
I think automethod will display the deprecation warning, so no need for an additional one?
![Screenshot 2021-12-08 at 12 53 11 AM](https://user-images.githubusercontent.com/30778939/145092918-42652b83-44bf-4da3-ad98-aa460740f5fc.png)

also needs some improvement for better display

Reply:
removed additional one